index.html

```diff
@@ -5,29 +5,30 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
     <title>Book List Web App</title>
 </head>
 
 <body>
 
-    <form id="booklist-form">
-
-        <h1>My Book List</h1>
+     <div class="layout-cont">
+        <h1 class="heading-primary">All awesome books</h1>
 
         <div id="book-shelf">
         
 
```

Comment:
- [ ] OPTIONAL
Kindly ensure not to add leave empty lines in your code which are not serving any purpose, so as to keep your code cleaner.